### PR TITLE
applying the patch to develop for master release - bobasca/etherscan

### PR DESCRIPTION
**Pull request by InoMurko**
inomurko/sahil-fix-udpate-blockexplore-url-to-bobascan

## Overview

With the release of Bobascan (Etherscan) fixing URLs in the Gateway and in Metamask.

## Changes

Pointing to Bobascan instead of our own Blockexplorer.

- Metamask config
- Urls in gateway

## Testing

/

### CODE REVIEW

File: packages/boba/gateway/src/util/masterConfig.js [hunk 1]
```diff
@@ -78,8 +78,8 @@ if (process.env.REACT_APP_CHAIN === 'rinkeby') {
         chainId: 288,
         chainIdHex: '0x120',
         rpcUrl: `https://mainnet.boba.network`,
-        blockExplorer: `https://blockexplorer.boba.network/`,
-        transaction: `https://blockexplorer.boba.network/tx/`,
+        blockExplorer: `https://bobascan.com/`,
+        transaction: `https://bobascan.com/tx/`,
```
Comment (souradeep-das) [inline packages/boba/gateway/src/util/masterConfig.js:82]:
```suggestion
        transaction: `https://bobascan.com/txs/`,
```

File: packages/boba/gateway/src/util/masterConfig.js [hunk 2]
```diff
@@ -37,8 +37,8 @@ if (process.env.REACT_APP_CHAIN === 'rinkeby') {
         chainId: 28,
         chainIdHex: '0x1C',
         rpcUrl: `https://rinkeby.boba.network`,
-        blockExplorer: `https://blockexplorer.rinkeby.boba.network/`,
-        transaction: `https://blockexplorer.rinkeby.boba.network/tx/`
+        blockExplorer: `https://testnet.bobascan.com/address/`,
+        transaction: `https://testnet.bobascan.com/address/tx/`
```
Comment (souradeep-das) [inline packages/boba/gateway/src/util/masterConfig.js]:
```suggestion
        blockExplorer: `https://testnet.bobascan.com/`,
        transaction: `https://testnet.bobascan.com/txs/`
```
maybe ?

**Review by souradeep-das — APPROVED** [review]

File: packages/boba/gateway/src/util/masterConfig.js (hunk 1 above)
Comment (InoMurko) [reply packages/boba/gateway/src/util/masterConfig.js:82]: is it tho? https://testnet.bobascan.com/tx/0x4983d0a97b8e0b6cbc3c3207cd9cf84fead1b79de22d5dceb65176c123de1d38

Comment (sk-enya) [reply packages/boba/gateway/src/util/masterConfig.js:82]: Seems `/tx` is correct one!

**Review by sk-enya — APPROVED** [review]

File: packages/boba/gateway/src/util/masterConfig.js (hunk 1 above)
Comment (souradeep-das) [reply packages/boba/gateway/src/util/masterConfig.js:82]: aah you guys are right!

**Pull request merged**